pages/manual/manual-gettingstarted-build.md

```diff
@@ -46,6 +46,16 @@ If you use a single-configuration generator, you can control the build mode usin
  Unless you specifically need to debug the application, we recommend setting the build mode to `RelWithDebInfo` for good performance, while still getting reasonable stacktraces for debugging and error reporting.
 " %}
 
+### Recommended setup (Optional)
+
+1. Build Type: Set the build mode to `RelWithDebInfo` ([See this guide for Visual Studio](https://docs.microsoft.com/en-us/visualstudio/debugger/how-to-set-debug-and-release-configurations?view=vs-2019))
+
+2. Build with multiple cores:
+    - Visual Studio: `Tools->Options->Projects and Solutions->Build and Run->maximum number of parallel project builds` We recommend 2-4 on an 8-core machine
+    - Qt Creator: In `Projects->Build & Run->Build->Build Steps->Details->CMake arguments` add `-j <number of cores>`, e.g. `-j4`
+
+3. Use multiple threads in Inviwo: In CMake set `IVW_MULTIPROCESSOR_COUNT` to the number of cores in your system.
```

Comment:
this has noting to to with how Inviwo uses thread, Is about how many threads that visual studio uses to build each project

Reply:
The instructions in step 3 should replace step 2?
I am also not sure how relevant these instructions are anymore as the IDEs might be a bit smarter nowadays?

Reply:
1 is for the number of parallel projects, and 3 is for the number of cl.exe processes per project so the total number of core used can be the product of 1 and 3. I have not used them for years, but then I have lots of cores, so In most cases my computer still stays responsive. Maybe @martinfalk still uses them? I would definitely __not__ set them by default, Only if I'm having issues with VS taking to may resources. At some point cl.exe also tended to crash for lack of memory (when running too many of them). But I have not seen that for a long time either.

Reply:
Updated and moved this to the note above. Removed "Recommended setup"

Reply:
I occasionally used this setting to ensure that my computer remains responsive during compilation by sparing one core.